app/helpers/application_helper.rb

```diff
@@ -1,7 +1,8 @@
 module ApplicationHelper
   def code_person_widget
     content_tag :div, id: "widget-code-person" do
-      image_tag random_person_image_url
+      image_tag random_person_image_url,
+        alt: 'Programming person with notebook'
```

Comment:
Could we change `notebook` to `laptop` here pls, as I would hear notebook to be a physical notebook (maybe a i8n difference?)

Reply:
Or even maybe "computer" as moving forward with might not always have laptops.

A question (for my edification): Is there a "correct" version of "Person programming on a computer" vs "Programming person with computer". ie, specifying a noun vs verb/action. I see a thing happening when I see those images, rather than a static image. But I have no idea what's correct or helpful when writing alt tags.

Reply:
Thanks for the note, English is not my mother tongue so I'm happy to get feedback on mistakes. For sure I'll change it to computer.

There're a few things to deal with when describing an image:
 - Worst case is to skip the alt attribute, an assistive technology like a screen-reader will then probably read out the filename :(
 - If the purpose of an image is only to support design, use an empty value `alt=""`, it will be ignored. Don't use aria here, it's meant to be used for special cases and it's most of the time better to use native HTML.
 - Briefly describe images to transport a similar information than you do visually. There's no need to go into details.
 - If the image holds some important content, like a chart does, you should try to describe it with more details. Though there are other strategies to use in such cases like creating interactive SVGs that support focusing on elements and description references, or simply adding visually hidden tables.